### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by gpmaccari** [lingxusb/megaDNA#1]
Hi thanks for sharing this project,
would it be possible for you to share the training script?
thanks in advance
G

**Comment by lingxusb** [comment]
Thank you for your interest in our work. The training process is pretty standard (please check our manuscript here: https://www.biorxiv.org/content/10.1101/2023.12.18.572218v3.full.pdf). I believe the training dataset plays a more crucial role, and we plan to share more detailed information about it soon. Please stay tuned.
Bin

**Comment by gpmaccari** [comment]
Thanks for the reply, it would be useful to have a training script anyways. I read with interest the manuscript and I understand you used MEGABYTE model as a base. Is the script used for training on this [torch porting](https://github.com/lucidrains/MEGABYTE-pytorch) of MEGABYTE similar to the one you used for training?

Some pointers would be useful.
Thanks again
G

**Comment by lingxusb** [comment]
Yes, it is very similar to that. Please check our manuscript for learning rate, tokenization, etc. (methods section)

Bin
